platform/viewer/src/routes/index.tsx

```diff
@@ -47,7 +46,19 @@ const createRoutes = ({
       hotkeysManager,
     }) || [];
 
-  const allRoutes = [...routes, ...bakedInRoutes];
+  const { uiCustomizationService } = servicesManager.services;
+
+  const customRoutes = uiCustomizationService.getGlobalCustomization(
+    'customRoutes'
+  );
+  const customBakedInRoutes =
+    customRoutes?.customizeBakedInRoutes?.(bakedInRoutes) || bakedInRoutes;
+
+  const allRoutes = [
+    ...routes,
+    ...(customRoutes?.routes || []),
+    ...customBakedInRoutes,
```

Comment:
what is the difference between .routes and customeBakedInRoutes, please add a comment